event_registration_analytic/models/event.py

```diff
@@ -32,6 +35,11 @@ class EventEvent(models.Model):
 
     @api.one
     @api.depends('registration_ids')
+    def _count_all_registrations(self):
+        self.count_all_registrations = len(self.registration_ids)
+
+    @api.one
```

Comment:
¿Por que están los campos calculados como api.one? No me había fijado en las que andabas subiendo últimamente, pero en principio se ponen api.multi y con un for.